src/styles/tailwind.css

```diff
@@ -1,3 +1,110 @@
+
+    /* Hovered elements */
+    --ob-bg-modifier-hover: var(--background-modifier-hover);
+    /*Active hovered elements*/
+    --ob-bg-modifier-active-hover: var(--background-modifier-active-hover);
+
+    /*Border color*/
+    --ob-bg-modifier-border: var(--background-modifier-border);
+    /*Border color (hover)*/
+    --ob-bg-modifier-border-hover: var(--background-modifier-border-hover);
+    /*Border color (focus)*/
+    --ob-bg-modifier-border-focus: var(--background-modifier-border-focus);
+
+    /*Error background, RGB value*/
+    --ob-bg-modifier-error-rgb: var(--background-modifier-error-rgb);
+    /*Error background*/
+    --ob-bg-modifier-error: var(--background-modifier-error);
+    /*Error background (hover)*/
+    --ob-bg-modifier-error-hover: var(--background-modifier-error-hover);
+
+    /*Success background, RGB value*/
+    --ob-bg-modifier-success-rgb: var(--background-modifier-success-rgb);
+    /*Success background*/
+    --ob-bg-modifier-success: var(--background-modifier-success);
+
+    /*Messages background*/
+    --ob-bg-modifier-message: var(--background-modifier-message);
+
+    /*Form field background*/
+    --ob-bg-modifier-form-field: var(--background-modifier-form-field);
+
+    /*Interactive colors*/
+    --ob-interactive-normal: var(
+      --interactive-normal
+    ); /* Background for standard interactive elements */
+    --ob-interactive-hover: var(
+      --interactive-hover
+    ); /*Background for standard interactive elements (hover)*/
+    --ob-interactive-accent: var(
+      --interactive-accent
+    ); /*Background for accented interactive elements*/
+    --ob-interactive-accent-hsl: var(
+      --interactive-accent-hsl
+    ); /* Background for accented interactive elements in HSL */
+    --ob-interactive-accent-hover: var(
+      --interactive-accent-hover
+    ); /*Background for accented interactive elements (hover)*/
+
+    /*Text foreground colors*/
+    --ob-text-normal: var(--text-normal); /*Normal text*/
+    --ob-text-muted: var(--text-muted); /*Muted text*/
+    --ob-text-faint: var(--text-faint); /*Faint text*/
+    --ob-text-on-accent: var(--text-on-accent); /*Text on accent background when accent is dark*/
+    --ob-text-on-accent-inverted: var(
+      --text-on-accent-inverted
+    ); /*Text on accent background when accent is light*/
+    --ob-text-success: var(--text-success); /*Success text*/
+    --ob-text-warning: var(--text-warning); /*Error text*/
+    --ob-text-error: var(--text-error); /*Error text*/
+    --ob-text-accent: var(--text-accent); /*Accent text*/
+    --ob-text-accent-hover: var(--text-accent-hover); /*Accent text (hover)*/
+
+    /*Text background colors*/
+    --ob-text-selection: var(--text-selection); /*Selected text background*/
+    --ob-text-highlight-bg: var(--text-highlight-bg); /*Highlighted text background*/
+  }
+}
+
+/* this layer for overwrite Obsidian default css style(file: app.css) and theme css style */
```

Comment:
I don't think we should be touching the default component style as it will affect UI outside of obsidian-copilot

Reply:
@zeroliu It doesn't affect UI outside of obsidian-copilot, It only work for copilot ui.

Reply:
This will affect all matching components. I created a sample plugin and added a button with "select-trigger-reset" class. By loading this version of copilot, my settings button now looks different because of this style set in copilot plugin:

<img width="955" alt="Screenshot 2024-12-21 at 9 04 53 PM" src="https://github.com/user-attachments/assets/718338de-32de-485d-b10e-b04fd515c551" />

Reply:
Just checked again, this is not a new problem introduced by this PR. The existing style can also affect other plugins. If I add `warning-message` to my plugin element and load the master version copilot, my element will also be affected. I sent a question to the obsidian discord to understand how style isolation works in obsidian. It seems very error-prone.

Reply:
`select-trigger-reset` is my custom css classname. Others have a very low probability of using it. I was aware of 
 css conflicts when developing the UI earlier.

> This will affect all matching components. 

The reason：
![CleanShot 2024-12-23 at 10 09 15](https://github.com/user-attachments/assets/9df93cd9-35d9-41aa-b135-488589838256)